media/css/mozorg/webvision.scss

```diff
@@ -0,0 +1,75 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/sidebar-menu';
+@import '~@mozilla-protocol/core/protocol/css/templates/main-with-sidebar';
```

Comment:
You're importing this template but I don't think you're currently using it for the layout. I think you can either adjust the HTML so the template gets used (it needs a few classes and wrapper elements) or you can remove this import if you're not using it.

Reply:
Also you have some classes for `mzp-c-article` but aren't importing the styles for the Article component. If you add the component styles you might be able lose some of the extra bits below.

`@import '~@mozilla-protocol/core/protocol/css/components/article';`

Reply:
@craigcook We merged the [CSS bundles PR](https://github.com/mozilla/bedrock/pull/11360) to change how we import some Protocol components (including `article`) so it's currently being imported here: https://github.com/mozilla/bedrock/pull/11387/files#diff-c9613986cc39c99ae415f69fb6c3baabe81e86e6772051fcdd5c7e2c0c19cc1aR12

Reply:
Ah, I missed that, didn't know we had done a bundle for article yet.

Reply:
Which rules specifically?

Reply:
Well ok, I was wrong again... Article used to have some font sizing rules for headings but they were removed in https://github.com/mozilla/protocol/commit/09ab37454d3189fbe778a82bf6dd38016462e917 and my memory is just faulty.